src/Api/Controllers/CiphersController.cs

```diff
@@ -94,8 +94,12 @@ public async Task<ListResponseModel<CipherDetailsResponseModel>> Get()
             Dictionary<Guid, IGrouping<Guid, CollectionCipher>> collectionCiphersGroupDict = null;
             if (hasOrgs)
             {
+                var keyMatches = ciphers.Select(c => c.Id); // Soft deletes not filtered in tweener "Get" methods
```

Comment:
I am a little confused about what we're doing here. Also, this should probaby be a HashSet for better perf

Reply:
This is intended to filter the collections for the ciphers returned (vs. all of them).. but this was a legacy change from my first commit I missed in reverting so I'll revert this.